backend/src/test/resources/tests/sql/selects/validity_date/multirange/expected.csv

```diff
@@ -0,0 +1,4 @@
+pid,datum
+1,"{[2012-01-01,2013-01-01),[2014-01-01,2015-01-01)}"
```

Comment:
```suggestion
1,"{[2012-01-01,2012-12-31),[2014-01-01,2014-31-12)}"
```

liegt aber vermutlich eher an der inkl/exkl darstellung :<

Reply:
Genau, lass uns dazu nochmal sprechen, es ist sehr frickelig das mit SQL in die gewünschte Darstellung zu bekommen - wir hatten daran gedacht, dass ins Post-Processing zu verschieben.

Reply:
Nochmal ein Hinweis dazu: ich habe in einem nächsten PR das Zurück-Casten in die korrekten Types schon vorbereitet, da wird dann auch die korrekt inkl./exkl. Darstellung verwendet.